commands.js

```diff
@@ -1490,6 +1491,7 @@ var commands = exports.commands = {
 		this.logModCommand("The ladder was disabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-red\"><b>Due to high server load, the ladder has been temporarily disabled</b><br />Rated games will no longer update the ladder. It will be back momentarily.</div>");
 	},
+	disableladderhelp: ["/disableladder - Disables the ladder, which will stop rated battles from being played and stop the ladder from updating. Requires: ~"],
```

Comment:
I would recommend not documenting this command at all. It's mainly a server load command and doesn't have use outside of PS Main.